platform-sdk/docs/core/dynamic-address-book/roster/glossary-of-terms.md

```diff
@@ -0,0 +1,148 @@
+maximum round that this node can gossip about.  Events which have a birth round higher than the `futureEventHorizon` 
+will not be gossipped.    
+
+### Past Event Horizon (a.k.a. Min Gossip Round)
+
+The `pastEventHorizon`, equal to `minRoundNonAncient` is the minimum round for which the node has an effective roster. 
+It is also the minimum round that this node can gossip about.  Events which have a birth round lower than the
+`pastEventHorizon` are ancient and will not be gossipped.
+
+### Validatable Event Window (within the Event Horizon)
+
+The `validatableEventWindow` is the window of rounds which have effective rosters that have not become ancient.  It 
+spans, inclusively, from the `pastEventHorizon` to the `futureEventHorizon`.  Events with `birthRound` within this 
+window are non-ancient and can be validated.
+
+## Roster Terminology
+
+### Roster (a.k.a. Consensus Roster)
+
+The collection of nodes which create events and participate in consensus.  Prior to the dynamic address book effort, 
+this was called the `AddressBook` in the platform's code.  The name was changed to `Roster` to disambiguate it from 
+the application's concept of an address book.  The application's address book contains the consensus roster.  
+
+### Effective Roster (for a round) 
+
+The roster that was used to determine consensus for a particular round.  The `effective roster for round X` is the 
+roster that was used to compute consensus for round X. 
+
+The effective roster is used for more than just consensus:
+* **Networking** - determine who to connect to and facilitate mutual TLS connection with that node. 
+* **Event Creation** - determine which other nodes' events are available to use as parents of events.
+* **Event Validation** - validate the signature using the signing key of the node that created the event.
+* **State Validation** - validating the signatures on the state of a round using the signatures in the effective 
+  roster for that round.
+
+### Roster's Effective Round
+The round that the roster will be used to calculate consensus for. This is the round number used to look up the 
+appropriate roster to validate an event. The event contains this round number as the event's `birthRound` and is 
+used to uniquely identify the roster that should be used to validate the event.
+
+Not a field inside the roster. 
+
+### Roster's Creation Round
+
+The round of transactions that created this roster once they were applied to the state. 
+
+### Roster Offset
+
+A setting.  The number of rounds it takes for a roster to become effective after being created. This is likely to be 
+a small integer value, e.g. 2. If this number is 2, then processing all the transactions in round 100 yields the 
+effective roster for round 102.
+
+### Ancient Roster
+
+All rosters that are effective for ancient rounds are ancient rosters. 
+
+### Minimum Roster Non-Ancient || Least Effective Roster
```

Comment:
I'm personally not a fan of "Least Effective Roster"

Reply:
I had it there as symmetry with `greatestEffectiveRoster` :: `futureEventHorizon`.   I'm open to name changes that make sense.  `pastEventHorizon` :: ???

Reply:
What about `earliestEffectiveRoster` or `oldestEffectiveRoster`? `leastEffectiveRoster` evokes a sense that the roster is somehow "less effective" than others 🤔

Reply:
I've removed the term and leave only `minRosterNonAncient`